api/server.py

```diff
@@ -1,8 +1,24 @@
-from flask import Flask, jsonify, request, abort
+from flask import Flask, jsonify, request, abort, session
 from flask_cors import CORS
+from flask.ext.session import Session
```

Comment:
flask.ext.session is deprecated. Use `from flask_session import Session` instead.